### ISSUE TITLE
Merging segments - Unexpected error: runtime error: slice bounds out of range [7:6]

### ISSUE THREAD

**Issue by mkstephenson** [ByteDream/crunchyroll-go#18]
Hi, just started trying to archive GATE and came across this error. The command I ran was `.\crunchy-v2.0.0_windows.exe archive -l all -o "S{episode_number}E{episode_number} - {title}.mkv" https://www.crunchyroll.com/gate` which produced the following log output:
```
=> Logged in
| en-US has no matching episodes
| es-419 has no matching episodes
| es-ES has no matching episodes
| fr-FR has no matching episodes
| pt-PT has no matching episodes
| pt-BR has no matching episodes
| it-IT has no matching episodes
| de-DE has no matching episodes
| ru-RU has no matching episodes
| ar-SA has no matching episodes
-> Parsed url 1
=> GATE Season 1
=>      1. The Self-Defense Force Goes to Another World » 1920x1080px, 23.97 FPS (S01E01)
=>      2. The Two Armies » 1920x1080px, 23.97 FPS (S01E02)
=>      3. Fire Dragon » 1920x1080px, 23.97 FPS (S01E03)
=>      4. To Unknown Lands » 1920x1080px, 23.97 FPS (S01E04)
=>      5. The Battle of Italica » 1920x1080px, 23.97 FPS (S01E05)
=>      6. Ride of the Valkyries » 1920x1080px, 23.97 FPS (S01E06)
=>      7. The Princess's Decision » 1920x1080px, 23.97 FPS (S01E07)
=>      8. Japan, Beyond the Gate » 1920x1080px, 23.97 FPS (S01E08)
=>      9. The Hakone Mountain Night Battle » 1920x1080px, 23.97 FPS (S01E09)
=>      10. Despair and Hope » 1920x1080px, 23.97 FPS (S01E10)
=>      11. Visitor » 1920x1080px, 23.97 FPS (S01E11)
=>      12. What Would Itami Do? » 1920x1080px, 23.97 FPS (S01E12)
=>      13. The Banquet Begins » 1920x1080px, 23.97 FPS (S01E13)
=>      14. The Imperial Capital Quake » 1920x1080px, 23.97 FPS (S01E14)
=>      15. Tuka Luna Marceau » 1920x1080px, 23.97 FPS (S01E15)
=>      16. The Fire Dragon, Once More » 1920x1080px, 23.97 FPS (S01E16)
=>      17. Decisive Battle » 1920x1080px, 23.97 FPS (S01E17)
=>      18. The Magic City of Rondel » 1920x1080px, 23.97 FPS (S01E18)
=>      19. Dangerous Sisters » 1920x1080px, 23.97 FPS (S01E19)
=>      20. Lover » 1920x1080px, 23.97 FPS (S01E20)
=>      21. Deadline » 1920x1080px, 23.97 FPS (S01E21)
=>      22. The Empress in Slave's Clothing » 1920x1080px, 23.97 FPS (S01E22)
=>      23. Paradrop » 1920x1080px, 23.97 FPS (S01E23)
=>      24. Thus They Fought » 1920x1080px, 23.97 FPS (S01E24)

=> Downloading episode `The Self-Defense Force Goes to Another World` to `S01E01 - The Self-Defense Force Goes to Another World (1).mkv` (merging audio for additional formats)
=>      Episode: S01E01
=>      Audio: ja-JP
=>      Subtitle:
=>      Resolution: 1920x1080px
=>      FPS: 23.97
=> Merging segments [=====================>]  99%      711/712=> Unexpected error: runtime error: slice bounds out of range [7:6]
```
Any idea why? I assume because it finishes downloading before it has all the expected segments, but why it does that is a bit puzzling.

OS: Windows 10 x64
Location: Germany

**Comment by mkstephenson** [comment]
Quick update - this works with the download commad, there I ran `.\crunchy-v2.0.0_windows.exe download -o "S{episode_number}E{episode_number} - {title}.ts" https://www.crunchyroll.com/gate` and it works fine

**Comment by ByteDream** [comment]
This is caused by [this](https://github.com/ByteDream/crunchyroll-go/blob/449cc3789636b33539ba5ef2168b924471e727a2/cmd/crunchyroll-go/cmd/archive.go#L393) line which is meant to removes subtitles. I don't know why I've built that in since all subtitles should be stored with the `archive` sub-command

**Comment by ByteDream** [comment]
Fix is available in [v2.0.1](https://github.com/ByteDream/crunchyroll-go/releases/tag/v2.0.1)
